src/migrate_db_to_postgresql.py

```diff
@@ -234,6 +237,9 @@ def _fix_illegal_dict(dict_: dict, label=''):  # pylint: disable=too-complex
             _fix_illegal_dict(value, label)
         elif isinstance(value, list):
             _fix_illegal_list(value, key, label)
+        elif isinstance(value, tuple):
+            dict_[key] = value = list(value)
```

Comment:
Looks wired, is this intended?

Reply:
Yes, `a = b = c` is just a short form of 
```python
a = c
b = c
```
and we need both (because reassigning `value` will not change `dict_[key]`)